src/hotspot/cpu/aarch64/frame_aarch64.cpp

```diff
@@ -468,7 +469,8 @@ frame frame::sender_for_compiled_frame(RegisterMap* map) const {
   // have to find it relative to the unextended sp
 
```

Comment:
The comment above this change needs to be updated to explain when and why it is correct to use 1) the unextended sp and frame size or 2) the sender sp.

Reply:
Thanks. 
I changed the comment, but this seems cannot be clearly explained in one or two sentences.
Please feel free to give any advice to refine the comment.

Reply:
Thanks. That's actually very clear and I don't think I can improve it.